### PR TITLE
Hiding prefixed variables in the debug variable panel

### PR DESCRIPTION
**Pull request by Christian-Holbrook**

### CODE REVIEW

File: src/LaunchConfiguration.ts [hunk 1]
```diff
@@ -175,6 +175,11 @@ export interface LaunchConfiguration extends DebugProtocol.LaunchRequestArgument
      * What level of debug server's internal logging should be performed in the debug session
      */
     logLevel: LogLevel;
+
+    /**
+     * Hide variables that are prefrixed with a special prefix designated to be hidden
+     */
+    hidePrefixVariables: boolean;
```
Comment (TwitchBronBron) [inline src/LaunchConfiguration.ts:182]: Let's name this `showHiddenVariables` instead. You'll also need to invert your logic everywhere. The default setting should be "hide the variables by default"

Comment (Christian-Holbrook) [reply src/LaunchConfiguration.ts:182]: Alright making that change.

**Review by TwitchBronBron — CHANGES_REQUESTED** [review]
A few small changes, but this looks great! Can you verify this works for local variables, as well as object properties?

What happens when I send a repl command like this?
```
> m.__rokudebug__value = true
> m.__rokudebug__value
```

**Review by TwitchBronBron — CHANGES_REQUESTED** [review]
Can you also add a few unit tests to make sure this works with the setting enabled and then disabled?

**Review by TwitchBronBron — APPROVED** [review]
Looks great! Thanks for the tests.

**Pull request merged**